src/content/language/functions.md

```diff
@@ -49,9 +49,13 @@ The <code>=> <em>expr</em></code> syntax is a shorthand for
 is sometimes referred to as _arrow_ syntax.
 
 :::note
-Only an *expression*—not a *statement*—can appear between the arrow (=\>) and
-the semicolon (;). For example, you can't put an [if statement][]
-there, but you can use a [conditional expression][].
+Only _expressions_ can appear between the arrow (=\>) and the semicolon (;).
+Expressions return values.
+This means that you can't write code in a place that expects a value.
+In the previous example,
+`_nobleGases[atomicNumber] != null;` returns a boolean value.
+The function then returns a value if the `atomicNumber` falls into
+the noble gas range. The function does nothing but return a value.
```

Comment:
I'd be careful with the term "return" as one might mistake it as "exit a function with a return value". Perhaps "evaluate" or "produce" might be a better term when referring to expressions?

Also, I think the distinction between an if statement and conditional expression is interesting. It shows an example of what you _can't_ use with the arrow syntax.

Here's a potential alternative:

```
Only _expressions_ can appear between the arrow (=\>) and the semicolon (;).
Expressions are anything that evaluate to a value.
For example, you could use a [conditional expression][] but not an [if statement][].
In the previous example,
`_nobleGases[atomicNumber] != null` is an expression that evaluates
to `true` if `_nobleGases` has an entry with the key `atomicNumber`.
```